vMenuServer/config/permissions.cfg

```diff
@@ -112,6 +112,12 @@ setr vmenu_current_minute 0
 # Enabling this will prevent you from setting a custom time, freezing the time and changing the time duration.
 setr vmenu_sync_to_machine_time "false"
 
+#####################################
```

Comment:
Unnecessary to have such a big comment block for this.
Also, as mentioned earlier, the naming scheme of this convar doesn't match the other convars.